README.md

```diff
@@ -46,6 +46,39 @@ if everything is working.
 
 ![screenshot](https://raw.githubusercontent.com/thaJeztah/pgadmin4-docker/master/pgadmin-screenshot.png)
 
+### Connect to a host running PostgreSQL
+
+To connect pgadmin4 with a database running in a host you have to modify two files in the host, pg-hba.conf and postgresql.conf
+
+Open the pg-hba.conf
+```bash
+#change 9.x to your PostgreSQL version
+$ sudo vim /etc/postgresql/9.x/main/pg_hba.conf
+```
+
+And add the line
+```conf
+host    all             all             0.0.0.0/0               trust
+```
+
+`0.0.0.0/0` means any host is allowed to connect, that can be replaced by the your container ip address
```

Comment:
"that can be replaced by the your container ip address"; looks like "your" should be removed; can you change to:

```Markdown
`0.0.0.0/0` means any host is allowed to connect. To restrict access to the pgAdmin container,
you can change this to the containers' IP-address.
```